adding-module.rst

```diff
@@ -116,16 +118,19 @@ add this code right after the includes:
 
 This creates a function object ``mymodule_hello_obj`` which takes no arguments,
 and when called, executes the C function ``mymodule_hello``. Also note, that
-our function has to return something -- so we return ``None``. Now we need to
+our function has to return something (as evey function returns an mp_obj_t-struct) -- so we return ``None``. Now we need to
```

Comment:
"every"
Please use a spellchecker before you submit a pull request.

It would be probably better to say "every Python function", because you can of course have functions of any type in C.